src/common/components/fileUploadField/FileUploadFieldFile.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { IconCheck, IconError } from "hds-react";
+
+import { FileUploadFieldProps } from "../../../domain/feedbackForm/types";
+import styles from "./fileUploadFieldFile.module.scss";
+
+function formatBytes(bytes: number, decimals = 2) {
+  const k = 1024;
+  const dm = decimals < 0 ? 0 : decimals;
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
```

Comment:
Great that you make this future-proof and take bigger file sizes into account 😄

Reply:
Do we have to make this translatable btw? bytes is now in English..

Reply:
Good point, I forgot to clean this one up. I think the actual form has a max file size defined in bits, so this one should be `k = 1000` as well.

I'll just do `Bytes` => `b` as this is only shown in error messages when the file is over the maximum size (5Mb).